src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+{
+    "APP.AUTH.LOGIN.BUTTON.REGISTER": "Go to registration",
+    "APP.AUTH.LOGIN.BUTTON.SUBMIT": "Sign in",
+    "APP.AUTH.LOGIN.LABEL.USERNAME": "Username:",
+    "APP.AUTH.LOGIN.LABEL.PASSWORD": "Password:",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.PASSWORD.REQUIRED": "Please enter your password to login.",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.MAXLENGTH": "Your entered username is too long and cannot be processed.",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.PATTERN": "Your entered username contains unsupported characters.",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.REQUIRED": "Please enter your username to login.",
+    "APP.AUTH.LOGIN.NOTIFICATION.HEADER": "Login",
+    "APP.AUTH.LOGIN.NOTIFICATION.MESSAGE.AUTH_INVALID_PASSWORD": "The entered password is wrong, please correct your input.",
```

Comment:
Password reset... actually a good idea for an improvement. How did we oversee that?